node/service/src/chain_spec/moonbase.rs

```diff
@@ -64,11 +64,14 @@ pub fn development_chain_spec(mnemonic: Option<String>, num_accounts: Option<u32
 				// Tech comitee members: Alith and Baltathar
 				vec![accounts[0], accounts[1]],
 				// Collator Candidate: Alice -> Alith
```

Comment:
```suggestion
				// Collator Candidates: Alice -> Alith, Bob -> Baltathar
```
update comment to include Bob

Reply:
I thin this change will not be kept because it means we couldn't run a single node `--dev` network. I'll either revert this entirely, or add a second spec with both authorities and the comment suggestion.

Reply:
I ended up reverting this because we can already `moonbeam --chain moonbase-local --dev-service`